<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB Grant Extractor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/archive-extract-2.ico</normaloff>:/icons/icons/archive-extract-2.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>DB Login Grantadmin:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Source Schema:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Target Schema:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Script Download Path:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdidConnection">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSource"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxTarget"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPath">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonLogin">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/network-connect-2.ico</normaloff>:/icons/icons/network-connect-2.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBrowsePath">
          <property name="text">
           <string>Script Pfad setzen</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icons/icons/system-search-3.ico</normaloff>:/icons/icons/system-search-3.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Download Script:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDownload">
        <property name="text">
         <string>Script Downloaden</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/icons/document-save-2.ico</normaloff>:/icons/icons/document-save-2.ico</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAktion">
    <property name="title">
     <string>Aktion</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionBrowsePath"/>
    <addaction name="actionScript_Donwnloaden"/>
    <addaction name="separator"/>
    <addaction name="actionCopyParam"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuAktion"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/application-exit-4.ico</normaloff>:/icons/icons/application-exit-4.ico</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/archive-extract-2.ico</normaloff>:/icons/icons/archive-extract-2.ico</iconset>
   </property>
   <property name="text">
    <string>Versionshinweis</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/network-connect-2.ico</normaloff>:/icons/icons/network-connect-2.ico</iconset>
   </property>
   <property name="text">
    <string>Login Grantadmin</string>
   </property>
  </action>
  <action name="actionBrowsePath">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/system-search-3.ico</normaloff>:/icons/icons/system-search-3.ico</iconset>
   </property>
   <property name="text">
    <string>Script Pfad setzen</string>
   </property>
  </action>
  <action name="actionScript_Donwnloaden">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/document-save-2.ico</normaloff>:/icons/icons/document-save-2.ico</iconset>
   </property>
   <property name="text">
    <string>Script Donwnloaden</string>
   </property>
  </action>
  <action name="actionCopyParam">
   <property name="text">
    <string>Parameter in Zwischenablage kopieren</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
